reliability-scanner/api/v1alpha1/service/annotations/README.md

```diff
@@ -0,0 +1,10 @@
+# Annotations
+
+checks for the existence of a specific key in service annotations
+
+Spec:
+```
```

Comment:
Was this supposed to be a markdown table?

Reply:
Yes, it was created using a markdown table generator. I have tested using Macdown. Let me know if you know of a better/alternate way?

Reply:
oh sorry. This was old. I will remove